src/UI/SafeStore.java

```diff
@@ -134,7 +134,7 @@ public static DebitCard getDebitCardInfo(String nickname) {
 
 	}
 	
-	public static boolean addDebitCard(String cardNumber, String nickname,String expDate, String cvv, String pin, String streetAddress, String city, String state, String zip) {
+	public static boolean addDebitCard(String cardNumber, String nickname, String expDate, String cvv, String pin, String streetAddress, String city, String state, String zip) {
```

Comment:
should the address be made before calling addDebitCard so that addDebitCard could take in 3 less parameters? 
```suggestion
	public static boolean addDebitCard(String cardNumber, String nickname, String expDate, String cvv, String pin, Address billingAddress) {
```